### DISCUSSION TITLE
Maps for regions in Germany

### DISCUSSION THREAD

**Discussion by mnoethe** [electricitymaps/electricitymaps-contrib#4987] [Ideas] [answered]
Is it possible to have co2 emissions detail for specific regions in Germany?

**Comment by VIKTORVAV99** [comment, marked as answer]
This has been brought up before and while we seem to be able to get production data for the grid regions we haven't found a good source to get the exchange data on the same granularity.

This has previously been discussed in #1216, #3795 and #4851.

So unfortunately we can't split them up right now but hopefully we will get more granular data in the future if one of the [ACER proposals](https://www.acer.europa.eu/sites/default/files/documents/Individual%20Decisions/ACER%20Decision%2011-2022%20on%20alternative%20BZ%20configurations.pdf) for Germany gets adopted. [Annex I (with maps).](https://www.acer.europa.eu/sites/default/files/documents/Individual%20Decisions_annex/ACER%20Decision%2011-2022%20on%20alternative%20BZ%20configurations%20-%20Annex%20I%20-%20rectified.pdf)

**Reply by mnoethe** [reply]
Thx for the feedback and the great app. 😀
